templates/events/event_form.html

```diff
@@ -0,0 +1,62 @@
+{% extends "base.html" %}
+{% load boxes %}
+
+{% block page_title %}Submit an Event | {{ SITE_INFO.site_name }}{% endblock %}
+{% block og_title %}Submit an Event{% endblock %}
+
+
+{% block content_attributes %}with-right-sidebar{% endblock %}
+
+
+
+{% block content %}
+    <article class="text">
+
+        <p>For bulk submittions, <a href="https://wiki.python.org/moin/PythonEventsCalendar#Submitting_an_Event">click here.</a></p>
+
+        <header class="article-header">
+            <h1 class="page-title">Submit an Event</h1>
+        </header>
+
+        <form class="wide-form" action="" method="post">
+            {% csrf_token %}
+            {% if next %}<input type="hidden" name="next" value="{{ next }}" />{% endif %}
+
+            {{ form.as_p }}
+            <p>
+                <button type="submit" name="Submit">Send</button>
+            </p>
+        </form>
+    </article>
+{% endblock content %}
+
+{% block right_sidebar %}
```

Comment:
It would be nice to avoid code duplication here since it was already defined in `templates/events/events_list.html`. I'd say create a new `templates/events/base.html` template and move all shared code there and change line 1 in this file from

```jinja
{% extends "base.html" %}
```

to

```jinja
{% extends "events/base.html" %}
```